.gitops/charts/traffic-court-online/charts/citizen-api/templates/deployment.yaml

```diff
@@ -47,6 +47,16 @@ spec:
             value: America/Vancouver
           - name: ASPNETCORE_URLS
             value: http://*:8080;http://*:9090
+          - name: COMS_USERNAME
+            valueFrom:
+             secretKeyRef:
```

Comment:
YAML is a bit tricky, the structure requires indenting 2 spaces like this:

![image](https://user-images.githubusercontent.com/1844480/216750557-8ca1448e-09e6-4463-8f22-438aea4a28d4.png)

Reply:
Fixed the YAML spacing.